### PR TITLE
Infra/ Move all colors and DesignTokens to public

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Move all colors and DesignTokens to public
Add `supportDarkMode` method to `Colors` class to allow enabling the dark mode tokens from private when the appearance changes 
WOAUILIB-2534

## Changelog
Move all colors and DesignTokens to public

### CODE REVIEW

File: src/style/colorsPalette.ts [hunk 1]
```diff
@@ -86,6 +86,8 @@ const colorsPalette = {
   purple70: '#f1daf7',
   purple80: '#faebfd',
   // VIOLETS
+  violet1: '#170E5C',
+  violet5: '#231690',
```
Comment (M-i-k-e-l) [inline src/style/colorsPalette.ts:90]: Where are these from?

File: src/style/colors.ts [hunk 2]
```diff
@@ -55,6 +56,15 @@ export class Colors {
     Scheme.setScheme(scheme);
   }
 
+  /**
+   * Support listening to Appearance changes
+   * and change the design tokens accordingly 
+   */
+  supportDarkMode(enable = false) {
```
Comment (M-i-k-e-l) [inline src/style/colors.ts:63]: This is not clear:
1. Why is there a default for `false`?
2. If you call this (I assume in private), then I am not sure about the name, why is it `support` and not `set`?

File: src/style/colorsPalette.ts (hunk 1 above)
Comment (lidord-wix) [reply src/style/colorsPalette.ts:90]: Alexey generated them :)

File: src/style/colors.ts (hunk 2 above)
Comment (lidord-wix) [reply src/style/colors.ts:63]: I changed it a little bit, no parameter needed, and I renamed it to `enableDarkMode`

File: src/style/colorsPalette.ts (hunk 1 above)
Comment (M-i-k-e-l) [reply src/style/colorsPalette.ts:90]: I did not see them in the docs 🤷‍♂️

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**